cardano-cli/cardano-cli.cabal

```diff
@@ -174,56 +174,97 @@ test-suite cardano-cli-test
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.ExtendedPaymentKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.GenesisDelegateKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.GenesisKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.GenesisUTxOKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.KESKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.PaymentKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.StakeKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Keys.VRFKeys
-                        Test.CLI.Shelley.Golden.TextEnvelope.Tx.Tx
-                        Test.CLI.Shelley.Golden.TextEnvelope.Tx.TxBody
-                        Test.CLI.Shelley.Golden.TextEnvelope.Tx.Witness
-                        Test.CLI.Shelley.Golden.TextView.DecodeCbor
-                        Test.CLI.Shelley.Golden.Transaction.Build
-                        Test.CLI.Shelley.Golden.Transaction.CalculateMinFee
-                        Test.CLI.Shelley.Golden.Transaction.Sign
-                        Test.CLI.Shelley.Tests
-                        Test.CLI.Version
-                        Test.ITN
+  other-modules:        Test.Cli.ITN
+                        Test.Cli.Pioneers.Exercise1
+                        Test.Cli.Pioneers.Exercise2
+                        Test.Cli.Pioneers.Exercise3
+                        Test.Cli.Pioneers.Exercise4
+                        Test.OptParse
+
+  default-language:     Haskell2010
+  default-extensions:   NoImplicitPrelude
+
+  ghc-options:          -Wall
+                        -Wincomplete-record-updates
+                        -Wincomplete-uni-patterns
+                        -Wredundant-constraints
+                        -Wpartial-fields
+                        -Wcompat
+                        -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-T
+
+  build-tool-depends:   cardano-cli:cardano-cli
+
+test-suite cardano-cli-golden
+  hs-source-dirs:       test
+  main-is:              cardano-cli-golden.hs
+  type:                 exitcode-stdio-1.0
+
+  build-depends:        base
+                      , aeson
+                      , bech32 >= 1.1.0
+                      , bytestring
+                      , base16-bytestring
+                      , cardano-api
+                      , cardano-cli
+                      , cardano-prelude
+                      , containers
+                      , deepseq
+                      , directory
+                      , hedgehog
+                      , lifted-base
+                      , optparse-applicative
+                      , process
+                      , temporary
+                      , text
+                      , time
+                      , transformers-except
+                      , unordered-containers
+
+  other-modules:        Test.Golden.Shelley
```

Comment:
I wonder if these should be `Test.Cli.Golden.*`.

Reply:
I specifically chopped the `Cli` part out because the test exe is `cardano-cli-golden`.

The other test in that package is `cardano-cli-test` .